msu/systems/persistent_data/persistent_data_mod_addon.nut

```diff
@@ -17,9 +17,14 @@
 	}
 
 	// MSU 1.3.0
+	function __prefixRaw()
```

Comment:
This only returns something so it should be `__getFilenamePrefix` and the other function which actually applies it should be `__prefixFilename`.